api/net/Areas/Editor/Controllers/ContentController.cs

```diff
@@ -203,6 +204,26 @@ public async Task<IActionResult> UnpublishAsync(ContentModel model)
         return new JsonResult(new ContentModel(result));
     }
 
+    /// <summary>
+    /// Request a transcript for the content for the specified 'id'.
+    /// Publish message to kafka to request a transcription.
+    /// </summary>
+    /// <param name="id"></param>
+    /// <returns></returns>
+    [HttpPut("{id}/transcribe")]
+    [Produces("application/json")]
+    [ProducesResponseType(typeof(ContentModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "Content" })]
+    public async Task<IActionResult> RequestTranscriptionAsync(long id)
```

Comment:
New API endpoint to requests transcriptions.   At some point we will need to capture a record of the request so that it can support other business requirements.